src/Common/src/Internal/Cryptography/BasicSymmetricCipher.cs

```diff
@@ -33,14 +33,22 @@ protected BasicSymmetricCipher(byte[] iv, int blockSizeInBytes)
 
         public int BlockSizeInBytes { get; private set; }
 
-        public virtual void Dispose()
+        public void Dispose()
         {
-            if (IV != null)
+            Dispose(true);
+            GC.SuppressFinalize(this);
```

Comment:
No, not yet.  I thought the idea was that you always put this call so when the day comes that a derived class has a finalizer, it is already taken care of.